### PR TITLE
docs: uniform spelling

### PR DESCRIPTION
**Pull request by vinckr**

### CODE REVIEW

File: docs/docs/ecosystem/deployment.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ accompanying Documentation.
 
 ## Data Storage and Persistence
 
-All ORY products support storing data in memory and in relational databases
+All ORY projects support storing data in memory and in relational databases
```
Comment (jfcurran) [inline docs/docs/ecosystem/deployment.md:15]:
```suggestion
All Ory projects support storing data in memory and in relational databases
```

File: docs/docs/ecosystem/deployment.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ to `memory`.
 
 ### SQL (persistent)
 
-All ORY products support PostgreSQL and MySQL as first-class citizens.
+All ORY projects support PostgreSQL and MySQL as first-class citizens.
```
Comment (jfcurran) [inline docs/docs/ecosystem/deployment.md:32]:
```suggestion
All Ory projects support PostgreSQL and MySQL as first-class citizens.
```

File: docs/docs/ecosystem/sqa.md [hunk 3]
```diff
@@ -11,22 +11,22 @@ stability of our open source service and identify potential issues.
 
 We are committed to full transparency on what data we transmit why and how. The
 source code of the telemetry package is completely open source and located
-[here](https://github.com/ory/x/blob/master/metricsx). If you do not wish to
-help us improve our products by sharing telemetry data, it is possible to
+[here](https://github.com/Ory/x/blob/master/metricsx). If you do not wish to
```
Comment (jfcurran) [inline docs/docs/ecosystem/sqa.md:14]:
```suggestion
[here](https://github.com/ory/x/blob/master/metricsx). If you do not wish to
```

File: docs/docs/ecosystem/sqa.md [hunk 4]
```diff
@@ -82,7 +82,7 @@ users. We are taking the following measures to protect your privacy:
    path parameters. A full list of transmitted URL paths is listed in section
    [Request telemetry](#request-telemetry).
 3. **We are unable to see or store the IP address of your host**, as the
-   [IP is set to `0.0.0.0`](https://github.com/ory/x/blob/master/metricsx/middleware.go)
+   [IP is set to `0.0.0.0`](https://github.com/Ory/x/blob/master/metricsx/middleware.go)
```
Comment (jfcurran) [inline docs/docs/ecosystem/sqa.md:85]:
```suggestion
   [IP is set to `0.0.0.0`](https://github.com/ory/x/blob/master/metricsx/middleware.go)
```

File: docs/docs/ecosystem/sqa.md [hunk 5]
```diff
@@ -129,12 +129,12 @@ a randomly assigned UUID v4 salt:
 
 ### Source Code
 
-The full code-base is [open source](https://github.com/ory/metrics-middleware).
+The full code-base is [open source](https://github.com/Ory/metrics-middleware).
```
Comment (jfcurran) [inline docs/docs/ecosystem/sqa.md:132]:
```suggestion
The full code-base is [open source](https://github.com/ory/metrics-middleware).
```

File: docs/docs/ecosystem/roadmap.md [hunk 6]
```diff
@@ -3,31 +3,31 @@ id: roadmap
 title: Roadmap
 ---
 
-## ORY Roadmap
+## Ory Roadmap
 
-If you would like a quick overview of whats coming next to ORY, you can do so at
-our [GitHub Projects Board](https://github.com/orgs/ory/projects/8).  
+If you would like a quick overview of whats coming next to Ory, you can do so at
+our [GitHub Projects Board](https://github.com/orgs/Ory/projects/8).
```
Comment (jfcurran) [inline docs/docs/ecosystem/roadmap.md:9]:
```suggestion
our [GitHub Projects Board](https://github.com/orgs/ory/projects/8).  
```

File: docs/docs/ecosystem/roadmap.md [hunk 7]
```diff
@@ -3,31 +3,31 @@ id: roadmap
...
-## ORY Project Management Board
+## Ory Project Management Board
 
-[To Do](https://github.com/orgs/ory/projects/8#column-6092768)  
-[Priority](https://github.com/orgs/ory/projects/8#column-7739379)  
-[In Progress](https://github.com/orgs/ory/projects/8#column-6092769)  
-[Review In Progress](https://github.com/orgs/ory/projects/8#column-6092770)  
-[Reviewer Approved](https://github.com/orgs/ory/projects/8#column-6092771)  
-[Done](https://github.com/orgs/ory/projects/8#column-6092772)
+[To Do](https://github.com/orgs/Ory/projects/8#column-6092768)
```
Comment (jfcurran) [inline docs/docs/ecosystem/roadmap.md:15]:
```suggestion
[To Do](https://github.com/orgs/ory/projects/8#column-6092768)  
```

File: docs/docs/ecosystem/roadmap.md [hunk 8]
```diff
@@ -3,31 +3,31 @@ id: roadmap
...
+## Ory Project Management Board
 
-[To Do](https://github.com/orgs/ory/projects/8#column-6092768)  
-[Priority](https://github.com/orgs/ory/projects/8#column-7739379)  
-[In Progress](https://github.com/orgs/ory/projects/8#column-6092769)  
-[Review In Progress](https://github.com/orgs/ory/projects/8#column-6092770)  
-[Reviewer Approved](https://github.com/orgs/ory/projects/8#column-6092771)  
-[Done](https://github.com/orgs/ory/projects/8#column-6092772)
+[To Do](https://github.com/orgs/Ory/projects/8#column-6092768)  
+[Priority](https://github.com/orgs/Ory/projects/8#column-7739379)
```
Comment (jfcurran) [inline docs/docs/ecosystem/roadmap.md:16]:
```suggestion
[Priority](https://github.com/orgs/ory/projects/8#column-7739379)  
```

File: docs/docs/ecosystem/roadmap.md [hunk 9]
```diff
@@ -3,31 +3,31 @@ id: roadmap
...
 
-[To Do](https://github.com/orgs/ory/projects/8#column-6092768)  
-[Priority](https://github.com/orgs/ory/projects/8#column-7739379)  
-[In Progress](https://github.com/orgs/ory/projects/8#column-6092769)  
-[Review In Progress](https://github.com/orgs/ory/projects/8#column-6092770)  
-[Reviewer Approved](https://github.com/orgs/ory/projects/8#column-6092771)  
-[Done](https://github.com/orgs/ory/projects/8#column-6092772)
+[To Do](https://github.com/orgs/Ory/projects/8#column-6092768)  
+[Priority](https://github.com/orgs/Ory/projects/8#column-7739379)  
+[In Progress](https://github.com/orgs/Ory/projects/8#column-6092769)
```
Comment (jfcurran) [inline docs/docs/ecosystem/roadmap.md:17]:
```suggestion
[In Progress](https://github.com/orgs/ory/projects/8#column-6092769)  
```

File: docs/docs/ecosystem/roadmap.md [hunk 10]
```diff
@@ -3,31 +3,31 @@ id: roadmap
...
-[To Do](https://github.com/orgs/ory/projects/8#column-6092768)  
-[Priority](https://github.com/orgs/ory/projects/8#column-7739379)  
-[In Progress](https://github.com/orgs/ory/projects/8#column-6092769)  
-[Review In Progress](https://github.com/orgs/ory/projects/8#column-6092770)  
-[Reviewer Approved](https://github.com/orgs/ory/projects/8#column-6092771)  
-[Done](https://github.com/orgs/ory/projects/8#column-6092772)
+[To Do](https://github.com/orgs/Ory/projects/8#column-6092768)  
+[Priority](https://github.com/orgs/Ory/projects/8#column-7739379)  
+[In Progress](https://github.com/orgs/Ory/projects/8#column-6092769)  
+[Review In Progress](https://github.com/orgs/Ory/projects/8#column-6092770)
```
Comment (jfcurran) [inline docs/docs/ecosystem/roadmap.md:18]:
```suggestion
[Review In Progress](https://github.com/orgs/ory/projects/8#column-6092770)  
```

File: docs/docs/ecosystem/roadmap.md [hunk 11]
```diff
@@ -3,31 +3,31 @@ id: roadmap
...
-[Priority](https://github.com/orgs/ory/projects/8#column-7739379)  
-[In Progress](https://github.com/orgs/ory/projects/8#column-6092769)  
-[Review In Progress](https://github.com/orgs/ory/projects/8#column-6092770)  
-[Reviewer Approved](https://github.com/orgs/ory/projects/8#column-6092771)  
-[Done](https://github.com/orgs/ory/projects/8#column-6092772)
+[To Do](https://github.com/orgs/Ory/projects/8#column-6092768)  
+[Priority](https://github.com/orgs/Ory/projects/8#column-7739379)  
+[In Progress](https://github.com/orgs/Ory/projects/8#column-6092769)  
+[Review In Progress](https://github.com/orgs/Ory/projects/8#column-6092770)  
+[Reviewer Approved](https://github.com/orgs/Ory/projects/8#column-6092771)
```
Comment (jfcurran) [inline docs/docs/ecosystem/roadmap.md:19]:
```suggestion
[Reviewer Approved](https://github.com/orgs/ory/projects/8#column-6092771)  
```

File: docs/docs/ecosystem/roadmap.md [hunk 12]
```diff
@@ -3,31 +3,31 @@ id: roadmap
...
-[In Progress](https://github.com/orgs/ory/projects/8#column-6092769)  
-[Review In Progress](https://github.com/orgs/ory/projects/8#column-6092770)  
-[Reviewer Approved](https://github.com/orgs/ory/projects/8#column-6092771)  
-[Done](https://github.com/orgs/ory/projects/8#column-6092772)
+[To Do](https://github.com/orgs/Ory/projects/8#column-6092768)  
+[Priority](https://github.com/orgs/Ory/projects/8#column-7739379)  
+[In Progress](https://github.com/orgs/Ory/projects/8#column-6092769)  
+[Review In Progress](https://github.com/orgs/Ory/projects/8#column-6092770)  
+[Reviewer Approved](https://github.com/orgs/Ory/projects/8#column-6092771)  
+[Done](https://github.com/orgs/Ory/projects/8#column-6092772)
```
Comment (jfcurran) [inline docs/docs/ecosystem/roadmap.md:20]:
```suggestion
[Done](https://github.com/orgs/ory/projects/8#column-6092772)
```

File: docs/docs/ecosystem/roadmap.md [hunk 13]
```diff
@@ -3,31 +3,31 @@ id: roadmap
...
 
 ## Individual Project Sections
 
-[ORY Kratos Milestones](https://github.com/ory/kratos/milestones)  
-[ORY Kratos Open Issues & Pull Requests](https://github.com/ory/kratos/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
+[Ory Kratos Milestones](https://github.com/Ory/kratos/milestones)
```
Comment (jfcurran) [inline docs/docs/ecosystem/roadmap.md:24]:
```suggestion
[Ory Kratos Milestones](https://github.com/ory/kratos/milestones)  
```

File: docs/docs/ecosystem/roadmap.md [hunk 14]
```diff
@@ -3,31 +3,31 @@ id: roadmap
...
+[Review In Progress](https://github.com/orgs/Ory/projects/8#column-6092770)  
+[Reviewer Approved](https://github.com/orgs/Ory/projects/8#column-6092771)  
+[Done](https://github.com/orgs/Ory/projects/8#column-6092772)
 
 ## Individual Project Sections
 
-[ORY Kratos Milestones](https://github.com/ory/kratos/milestones)  
-[ORY Kratos Open Issues & Pull Requests](https://github.com/ory/kratos/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
+[Ory Kratos Milestones](https://github.com/Ory/kratos/milestones)  
+[Ory Kratos Open Issues & Pull Requests](https://github.com/Ory/kratos/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
```
Comment (jfcurran) [inline docs/docs/ecosystem/roadmap.md:25]:
```suggestion
[Ory Kratos Open Issues & Pull Requests](https://github.com/ory/kratos/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
```

File: docs/docs/ecosystem/roadmap.md [hunk 15]
```diff
@@ -3,31 +3,31 @@ id: roadmap
...
 
-[ORY Hydra Milestones](https://github.com/ory/hydra/milestones)  
-[ORY Hydra Open Issues & Pull Requests](https://github.com/ory/hydra/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)  
-[ORY Keto Milestones](https://github.com/ory/keto/milestones)  
-[ORY Keto Open Issues & Pull Requests](https://github.com/ory/keto/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
+[Ory Hydra Milestones](https://github.com/Ory/hydra/milestones)
```
Comment (jfcurran) [inline docs/docs/ecosystem/roadmap.md:27]:
```suggestion
[Ory Hydra Milestones](https://github.com/ory/hydra/milestones)  
```

File: docs/docs/ecosystem/roadmap.md [hunk 16]
```diff
@@ -3,31 +3,31 @@ id: roadmap
...
-[ORY Kratos Open Issues & Pull Requests](https://github.com/ory/kratos/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
+[Ory Kratos Milestones](https://github.com/Ory/kratos/milestones)  
+[Ory Kratos Open Issues & Pull Requests](https://github.com/Ory/kratos/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
 
-[ORY Hydra Milestones](https://github.com/ory/hydra/milestones)  
-[ORY Hydra Open Issues & Pull Requests](https://github.com/ory/hydra/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)  
-[ORY Keto Milestones](https://github.com/ory/keto/milestones)  
-[ORY Keto Open Issues & Pull Requests](https://github.com/ory/keto/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
+[Ory Hydra Milestones](https://github.com/Ory/hydra/milestones)  
+[Ory Hydra Open Issues & Pull Requests](https://github.com/Ory/hydra/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
```
Comment (jfcurran) [inline docs/docs/ecosystem/roadmap.md:28]:
```suggestion
[Ory Hydra Open Issues & Pull Requests](https://github.com/ory/hydra/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)  
```

File: docs/docs/ecosystem/roadmap.md [hunk 17]
```diff
@@ -3,31 +3,31 @@ id: roadmap
...
+[Ory Kratos Milestones](https://github.com/Ory/kratos/milestones)  
+[Ory Kratos Open Issues & Pull Requests](https://github.com/Ory/kratos/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
 
-[ORY Hydra Milestones](https://github.com/ory/hydra/milestones)  
-[ORY Hydra Open Issues & Pull Requests](https://github.com/ory/hydra/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)  
-[ORY Keto Milestones](https://github.com/ory/keto/milestones)  
-[ORY Keto Open Issues & Pull Requests](https://github.com/ory/keto/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
+[Ory Hydra Milestones](https://github.com/Ory/hydra/milestones)  
+[Ory Hydra Open Issues & Pull Requests](https://github.com/Ory/hydra/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)  
+[Ory Keto Milestones](https://github.com/Ory/keto/milestones)
```
Comment (jfcurran) [inline docs/docs/ecosystem/roadmap.md:29]:
```suggestion
[Ory Keto Milestones](https://github.com/ory/keto/milestones)  
```

File: docs/docs/ecosystem/roadmap.md [hunk 18]
```diff
@@ -3,31 +3,31 @@ id: roadmap
...
+[Ory Kratos Open Issues & Pull Requests](https://github.com/Ory/kratos/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
 
-[ORY Hydra Milestones](https://github.com/ory/hydra/milestones)  
-[ORY Hydra Open Issues & Pull Requests](https://github.com/ory/hydra/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)  
-[ORY Keto Milestones](https://github.com/ory/keto/milestones)  
-[ORY Keto Open Issues & Pull Requests](https://github.com/ory/keto/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
+[Ory Hydra Milestones](https://github.com/Ory/hydra/milestones)  
+[Ory Hydra Open Issues & Pull Requests](https://github.com/Ory/hydra/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)  
+[Ory Keto Milestones](https://github.com/Ory/keto/milestones)  
+[Ory Keto Open Issues & Pull Requests](https://github.com/Ory/keto/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
```
Comment (jfcurran) [inline docs/docs/ecosystem/roadmap.md:30]:
```suggestion
[Ory Keto Open Issues & Pull Requests](https://github.com/ory/keto/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
```

File: docs/docs/ecosystem/roadmap.md [hunk 19]
```diff
@@ -3,31 +3,31 @@ id: roadmap
...
 
-[ORY Oathkeeper Milestones](https://github.com/ory/oathkeeper/milestones)  
-[ORY Oathkeeper Open Issues & Pull Requests](https://github.com/ory/oathkeeper/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
+[Ory Oathkeeper Milestones](https://github.com/Ory/oathkeeper/milestones)
```
Comment (jfcurran) [inline docs/docs/ecosystem/roadmap.md:32]:
```suggestion
[Ory Oathkeeper Milestones](https://github.com/ory/oathkeeper/milestones)  
```

File: docs/docs/ecosystem/roadmap.md [hunk 20]
```diff
@@ -3,31 +3,31 @@ id: roadmap
...
-[ORY Keto Open Issues & Pull Requests](https://github.com/ory/keto/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
+[Ory Hydra Milestones](https://github.com/Ory/hydra/milestones)  
+[Ory Hydra Open Issues & Pull Requests](https://github.com/Ory/hydra/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)  
+[Ory Keto Milestones](https://github.com/Ory/keto/milestones)  
+[Ory Keto Open Issues & Pull Requests](https://github.com/Ory/keto/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
 
-[ORY Oathkeeper Milestones](https://github.com/ory/oathkeeper/milestones)  
-[ORY Oathkeeper Open Issues & Pull Requests](https://github.com/ory/oathkeeper/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
+[Ory Oathkeeper Milestones](https://github.com/Ory/oathkeeper/milestones)  
+[Ory Oathkeeper Open Issues & Pull Requests](https://github.com/Ory/oathkeeper/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
```
Comment (jfcurran) [inline docs/docs/ecosystem/roadmap.md:33]:
```suggestion
[Ory Oathkeeper Open Issues & Pull Requests](https://github.com/ory/oathkeeper/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
```

**Review by jfcurran — APPROVED** [review]
👍

**Comment by aeneasr** [comment]
Please use batch commit for suggestions, or everyone watching will be spammed with notifications.

**Pull request merged**